Frends.Community.AWS/Utils.cs

```diff
@@ -1,9 +1,9 @@
-﻿using System.IO;
-using System.Threading;
-using System.Threading.Tasks;
-using Amazon;
+﻿using Amazon;
 using Amazon.S3;
 using Amazon.S3.Model;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
 
 namespace Frends.Community.AWS
 {

```

Comment:
https://github.com/CommunityHiQ/Frends.Community.AWS/blob/fef2efc654cb3383bd049a73b8d3b3d007c4078b/Frends.Community.AWS/Utils.cs#L80
This method is not used, so it can be removed.